demo-app/src/main.rs

```diff
@@ -0,0 +1,253 @@
+use std::marker::PhantomData;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use example_election::Election;
+// Items that should go in prelude
+use jmt::SimpleHasher;
+use sov_modules_api::{
+    mocks::{MockContext, MockPublicKey},
+    Context, DispatchCall, DispatchQuery, Genesis, Module, Spec,
+};
+use sov_modules_macros::{DispatchCall, DispatchQuery, Genesis, MessageCodec};
+use sov_state::{JmtStorage, Storage};
+use sovereign_sdk::{
+    core::{
+        mocks::MockProof,
+        traits::{BatchTrait, CanonicalHash, TransactionTrait},
+    },
+    jmt,
+    serial::Decode,
+    stf::{ConsensusSetUpdate, OpaqueAddress, StateTransitionFunction},
+};
+
+// we should re-export anyhow from the modules_api
+
+#[derive(Debug, PartialEq, Eq, Clone, BorshDeserialize, BorshSerialize)]
+pub struct DemoTransaction<C: Context> {
```

Comment:
non-blocking: in `basic_runtime.rs` we have the `Message` type. I think we should unify DemoTransaction and Message let's add TODO for it.